src/include/sof/audio/buffer.h

```diff
@@ -49,6 +49,10 @@ extern struct tr_ctx buffer_tr;
 /** \brief Retrieves subid (comp id) from the buffer */
 #define trace_buf_get_subid(buf_ptr) ((buf_ptr)->id)
 
+#if CONFIG_IPC_MAJOR_4
+#define SOF_SMART_AMP_FEEDBACK_QUEUE_ID        1
```

Comment:
ah, once I wanted to move the feedback buffer clear operate in smart_amp trigger to module_adpater triger ops, but since you explained it is not necessary in your comment below, so I will move it back to smart_amp header file.